app/Http/Controllers/PurchaseOrderController.php

```diff
@@ -15,7 +15,7 @@ class PurchaseOrderController extends Controller
      */
     public function index(Request $request)
     {
-        $statuses = PurchaseOrder::PENDING . "," . PurchaseOrder::RECEIVED . "," . PurchaseOrder::CANCELLED;
+        $statuses = PurchaseOrder::PENDING . "," . PurchaseOrder::RECEIVED . "," . PurchaseOrder::CANCELLED . "," . PurchaseOrder::PAID;
```

Comment:
To make this a proper refactor, i'd suggest you instead extract this to a private function on  the controller